pmdksummit2021/index.html

```diff
@@ -58,8 +58,8 @@ <h1 class="text-center title" id="portfolio">2021 PMDK Summit</h1>
 				<div class="separator"></div>
 				<p>
 				<b>Persistent Memory Development Kit (PMDK)</b><br>
-PMDK an open-source project hosted by Intel.  The introduction of Persistent Memory (PMEM) marks the beginning of a revolution in the computing industry.  
-<br>PMDK development kit is a collection of open-source (vendor-neutral) libraries that makes this new paradigm of merging Memory and Storage possible.  
+PMDK an open-source project hosted by Intel.  The introduction of Persistent Memory (PMEM) marks the beginning of a revolution in the computing industry.
+<br>This development kit is a collection of open-source (vendor-neutral) libraries that makes this new paradigm of merging Memory and Storage possible.
```

Comment:
PMDK is a ...

(I know it's a repetition, but it's better than using this)

Reply:
Done.